### DISCUSSION TITLE
Design Bug

### DISCUSSION THREAD

**Discussion by ALIRAZA47** [community/community#51508] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/57589739/228904845-fdaf9614-6b93-4f76-9b27-b038de0ce134.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ALIRAZA47 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
